pkgs/top-level/all-packages.nix

```
@@ -2980,6 +2980,8 @@ with pkgs;
 
   clac = callPackage ../tools/misc/clac {};
 
+  map_cmd = callPackage ../tools/misc/map_cmd {};
```

```suggestion
  map-cli = callPackage ../tools/misc/map {};
```
or map-xargs or map-cmd